lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -178,6 +178,22 @@ def assertThreadCorrect(thread, discussion, expected_title):  # pylint: disable=
         assertThreadCorrect(threads[0], self.discussion1, "Chapter / Discussion 1")
         assertThreadCorrect(threads[1], self.discussion2, "Subsection / Discussion 2")
 
+    def test_empty_discussion_subcategory_title(self):
+        """
+        Test that for empty subcategory inline discussion modules,
+        the divider " / " is not rendered on a post or inline discussion topic label.
+        """
+        discussion = ItemFactory.create(
+            parent_location=self.course.location,
+            category="discussion",
+            discussion_id="discussion",
+            discussion_category="Chapter",
+            discussion_target=""  # discussion-subcategory
+        )
+        thread = {"commentable_id": discussion.discussion_id}
+        utils.add_courseware_context([thread], self.course, self.user)
+        self.assertNotIn(thread.get("courseware_title"), ' / ')
```

Comment:
Additionally, one of my concerns was the spaces, so can we just look for `/`?  I know it is accurate as written, but a little brittle because `Category /` would pass this test.  Thanks.